DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -68,6 +71,7 @@ protocol NewWindowPolicyDecisionMaker {
 
     let startupPreferences: StartupPreferences
     let tabsPreferences: TabsPreferences
+    let phishingState: PhishingTabStateManager = PhishingTabStateManager()
```

Comment:
Yes it should actually, good spot. I think we might be able to make it private, perhaps if we can rely on `tab.privacyInfo.isPhishing` for the state instead of `phishingState` [here](https://github.com/duckduckgo/macos-browser/blob/4a222adcac8553ed0b51ce48d0bbdf868ecb185e/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift)